user_story/model/user_story.py

```diff
@@ -616,6 +618,14 @@ def _get_user_story_field(self, cr, uid, ids, fieldname, arg,
             res[ac_brw.id] = copy_field
         return res
 
+    def _get_user_story_state(self, cr, uid, ids, fieldname, arg,
+                              context=None):
+        context = {} if context is None else context
```

Comment:
Indeed, I saw that you aren't use the `context` then... you can delete this line.